lms/urls.py

```diff
@@ -434,6 +434,15 @@
             url(r'^courses/{}/teams'.format(settings.COURSE_ID_PATTERN), include('teams.urls'), name="teams_endpoints"),
         )
 
+    if settings.FEATURES.get('ENABLE_MOBILE_XBLOCKS'):
```

Comment:
Since this is intended to be the generic endpoint for LTI, Mobile, or whatever else needs chromeless XBlocks, can we rename the feature flag?